.github/workflows/release-candidate.yaml

```diff
@@ -15,121 +15,90 @@ env:
   RC_BRANCH_NAME: ${{ github.event.inputs.version }}-${{ github.event.inputs.date }}
 
 jobs:
-  check_docs:
-    name: Check docs build
+  docs:
+    name: Docs
     runs-on: ubuntu-22.04
     steps:
-      - name: Trigger repository build workflow
-        uses: convictional/trigger-workflow-and-wait@v1.6.1
+      - name: Check docs build
+        uses: convictional/trigger-workflow-and-wait@v1.6.5
         with:
           owner: husarion
           repo: docs_new
-          github_token: ${{ secrets.GH_PAT }}
-          workflow_file_name: test-release.yml
+          github_token: ${{ secrets.RAFAL_ACCESS_TOKEN }}
+          workflow_file_name: parser-test.yaml
           ref: master
           client_payload: '{"husarion_ugv_branch": "ros2-devel"}'
 
-  # TODO: Add unit testing for panther_ros when ready
-  unit_test_panther_ros:
-    name: Run unit tests for panther_ros
+  # TODO: Add unit testing when ready
+  unit_tests:
+    name: Unit tests
     runs-on: ubuntu-22.04
     steps:
-      - name: Trigger repository build workflow
-        run: echo "Unit tests for panther_ros are not fully implemented yet -> SKIPPING!"
+      - name: Run unit tests
+        run: echo "Unit tests are not fully implemented yet -> SKIPPING!"
 
-  create_release_candidate_branches:
-    name: Create release candidate branches
```

Comment:
release candidate branches should be created for both `panther_ros` and `panther-rpi-os-img` repos. Only, then the docker tag should be updated (on the new release branch).